superset/config.py

```diff
@@ -1427,11 +1431,6 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 SLACK_API_TOKEN: Callable[[], str] | str | None = None
 SLACK_PROXY = None
 
-# Whether Superset should use Slack avatars for users.
-# If on, you'll want to add "https://avatars.slack-edge.com" to the list of allowed
-# domains in your TALISMAN_CONFIG
-SLACK_ENABLE_AVATARS = False
-
```

Comment:
Since this hasn't been released in an official version, I think this breaking change is ok.

Reply:
Right, and I added a note in UPDATING.md as a heads-up